app/v2/services/AmendForeignPropertyPeriodSummaryService.scala

```diff
@@ -63,4 +63,13 @@ class AmendForeignPropertyPeriodSummaryService @Inject()(connector: AmendForeign
       "SERVER_ERROR" -> InternalError,
       "SERVICE_UNAVAILABLE" -> InternalError
     )
+
```

Comment:
Have you got your Scala formatter on? This hasn't been formatted  - can show you how to turn it on if we call